tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+                description: List of all the active subscription IDs
+                type: array
+                items:
+                  type: string
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/subscriptions/{id}:
+    get:
+      tags:
+        - subscriptions
+      summary: Get a subscription
+      description: Returns a subscription's metadata
+      operationId: GetSubscription
+      parameters:
+        - in: path
+          name: id
+          description: Subscription ID
+          type: string
+          required: true
+      responses:
+        200:
+          schema:
+            $ref: '#/definitions/Subscription'
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Subscription with the given ID was not found
+        default:
+          $ref: '#/responses/Error'
+    delete:
+      tags:
+        - subscriptions
+      summary: Delete a subscription
+      description: Deletes a specific subscription.
+      operationId: DeleteSubscription
+      parameters:
+        - in: path
+          name: id
+          description: Subscription ID
+          type: string
+          required: true
+      responses:
+        204:
+          description: Subscription successfully deleted
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Subscription with the given ID was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/subscriptions/{id}/heartbeat:
+    put:
+      tags:
+        - subscriptions
+      summary: Update the heartbeat timer for a subscription
+      description: Updates the heartbeat timer for the specified subscription. If a subscription's heartbeat timer elapses, the subscription will go inactive and will no longer record updates.
+      operationId: UpdateSubscriptionHeartbeat
+      parameters:
+        - in: path
+          name: id
+          description: Subscription ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+        401:
+          $ref: '#/responses/Unauthorized'
+        404:
+          description: Subscription with the given ID was not found
+        default:
+          $ref: '#/responses/Error'
+  /v2/subscriptions/{id}/values/current:
+    get:
+      tags:
+        - subscriptions
+      summary: Get the current values for the tags in a subscription
+      description: Returns the current values for the tags in the subscription. Can return either all values or only new values since the last request.
+      operationId: GetSubscriptionTagValues
+      parameters:
+        - in: path
+          name: id
+          description: Subscription ID
+          type: string
+          required: true
+      responses:
+        200:
+          schema:
+            $ref: '#/responses/GetSubscriptionUpdatesResponse'
```

Comment:
This is a response ref, so it needs to go up a level instead of under the schema section.